src/neg_tests/lin_tests_thread_ref.ml

```diff
@@ -2,12 +2,15 @@ open Lin_tests_common
 
 (** This is a driver of the negative ref tests over the Thread module *)
 
+module RT_int_thread = Lin_thread.Make_internal(RConf_int)
+module RT_int64_thread = Lin_thread.Make_internal(RConf_int64)
+
 ;;
 if Sys.backend_type = Sys.Bytecode
 then
   Printf.printf "Lin ref tests with Thread disabled under bytecode\n\n%!"
 else
 QCheck_base_runner.run_tests_main
   (let count = 1000 in
-   [RT_int.lin_test       `Thread ~count ~name:"Lin ref int test with Thread"; (* unboxed, hence no allocations to trigger context switch *)
-    RT_int64.neg_lin_test `Thread ~count:15000 ~name:"Lin ref int64 test with Thread"])
+   [RT_int64_thread.lin_test     ~count ~name:"Lin ref int test with Thread"; (* unboxed, hence no allocations to trigger context switch *)
```

Comment:
I think this was changed to an `int64` test by mistake - which explains why this test started failing consistently in the CI.

Reply:
Fixed in 5b0d03d